scripts/build_wasi.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# We download the SDK
+if [[ ! -d wasi-sdk-5.0 ]]; then
+    OS=$(uname | tr '[:upper:]' '[:lower:]')
+    case "$OS" in
+        darwin) WASI_URL='https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-5/wasi-sdk-5.0-macos.tar.gz';;
```

Comment:
How about using version 6?

Reply:
It's not available for mac 🙁 (only linux)

Reply:
Looks like it just got added: https://github.com/CraneStation/wasi-sdk/issues/59

Reply:
Awesome!